yt_dlp/extractor/tencent.py

```diff
@@ -63,7 +70,9 @@ def _get_video_api_response(self, video_url, video_id, series_id, subtitle_forma
 
     def _extract_video_formats_and_subtitles(self, api_response, video_id):
         video_response = api_response['vl']['vi'][0]
-        video_width, video_height = video_response.get('vw'), video_response.get('vh')
+        has_drm, video_width, video_height, video_format = video_response.get('drm') == 1, video_response.get('vw'), video_response.get('vh'), video_response.get('br')
+        format_response = next(filter(lambda f: f.get('br') == video_format, api_response['fl']['fi']))
+        abr, vbr, is_hdr, format_name = format_response.get('audiobandwidth') / 1000, format_response.get('bandwidth') / 1000, format_response.get('name') == 'hdr10', f'{format_response.get("sname")} ({format_response.get("resolution")})'
```

Comment:
This will fail if the field doesn't exist, use instead `float_or_none(format_response.get('audiobandwidth'), scale=1000)`

https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#mandatory-and-optional-metafields

```bash
[debug] Command-line config: ['https://wetv.vip/en/play/7csb8h9c79p8idi-Lie%20To%20Love', '--verbose', '-F']
[debug] User config: []
[debug] System config: []
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.02 [d83b0ad80] (source)
[debug] Lazy loading extractors is disabled
[debug] Git HEAD: f49f600fe
[debug] Python 3.10.7 (CPython x86_64 64bit) - Linux-5.19.0-26-generic-x86_64-with-glibc2.36 (OpenSSL 3.0.5 5 Jul 2022, glibc 2.36)
[debug] exe versions: ffmpeg N-108931-g4dda3b1653-20221104 (setts), ffprobe N-108931-g4dda3b1653-20221104, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.15.0, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1756 extractors
[WeTvSeries] Extracting URL: https://wetv.vip/en/play/7csb8h9c79p8idi-Lie%20To%20Love
[WeTvSeries] 7csb8h9c79p8idi: Downloading webpage
[download] Downloading playlist: Lie To Love
[WeTvSeries] Playlist Lie To Love: Downloading 32 items of 32
[download] Downloading item 1 of 32
[wetv:episode] Extracting URL: https://wetv.vip/play/7csb8h9c79p8idi/a0041yw44yk
[wetv:episode] a0041yw44yk: Downloading webpage
[wetv:episode] a0041yw44yk: Downloading webpage
ERROR: unsupported operand type(s) for /: 'NoneType' and 'int'
Traceback (most recent call last):
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/YoutubeDL.py", line 1502, in wrapper
    return func(self, *args, **kwargs)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/YoutubeDL.py", line 1578, in __extract_info
    ie_result = ie.extract(url)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/extractor/common.py", line 680, in extract
    ie_result = self._real_extract(url)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/extractor/tencent.py", line 381, in _real_extract
    return self._extract_episode(url)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/extractor/tencent.py", line 301, in _extract_episode
    formats, subtitles = self._extract_all_video_formats_and_subtitles(url, video_id, series_id)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/extractor/tencent.py", line 132, in _extract_all_video_formats_and_subtitles
    fmts, subs = self._extract_video_formats_and_subtitles(api_response, video_id)
  File "/home/amish/Documents/Programming/yt-dlp/yt_dlp/extractor/tencent.py", line 75, in _extract_video_formats_and_subtitles
    abr, vbr, is_hdr, format_name = format_response.get('audiobandwidth') / 1000, format_response.get('bandwidth') / 1000, format_response.get('name') == 'hdr10', f'{format_response.get("sname")} ({format_response.get("resolution")})'
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```

Reply:
Ok, I'll fix it when I wake up.